modules/database/main.tf

```diff
@@ -34,6 +34,7 @@ resource "aws_db_instance" "pg_db" {
   performance_insights_enabled = true
   storage_encrypted            = true
   backup_retention_period      = var.rds_db_retention_period
+  multi_az                     = var.rds_multi_az
```

Comment:
@ckluy31 you'll need to wire this up in the `main.tf` file and expose this variable in the root `variables.tf` so that it can be configured. You can take a look at `dynamodb_restore_to_latest_time` as an example of a variable which is wired up to the main module